### PR TITLE
feat(ci): disable cluster upgrade test

### PR DESCRIPTION
**Pull request by leoryu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e_installer/cluster/cluster_test.go [hunk 1]
```diff
@@ -108,19 +108,19 @@ var _ = Describe("cluster", func() {
...
+	// DescribeTable("Upgrade cluster",
+	// 	func(oldVersion, newVersion string) {
+	// 		cls = testTKE.ClusterTemplate()
+	// 		cls.Spec.Version = oldVersion
+	// 		cls, err = testTKE.CreateClusterInternal(cls)
+	// 		Expect(err).To(BeNil(), "Create cluster failed")
+
+	// 		cls, err = testTKE.UpgradeCluster(cls.Name, newVersion, platformv1.UpgradeModeAuto, false)
+	// 		Expect(err).Should(BeNil(), "Upgrade cluster failed")
+	// 		Expect(cls.Spec.Version).Should(Equal(newVersion), "Cluster version is wrong")
```
Comment (huxiaoliang) [inline test/e2e_installer/cluster/cluster_test.go:120]: I perfer to use "/* */" to comments block of code

**Review by huxiaoliang — CHANGES_REQUESTED** [review]
please address my comments

File: test/e2e_installer/cluster/cluster_test.go (hunk 1 above)
Comment (leoryu) [reply test/e2e_installer/cluster/cluster_test.go]: done

**Review by huxiaoliang — APPROVED** [review]

**Review by zhangzhangzf — APPROVED** [review]
lgtm

**Pull request merged**